docs/command-reference/tanzu_apps_workload_get.md

```diff
@@ -19,7 +19,7 @@ tanzu apps workload get my-workload
 ### Options
 
 ```
-      --export           export workload in yaml format
+  -e, --export           export workload in yaml format
```

Comment:
please update the [detail doc](https://github.com/vmware-tanzu/apps-cli-plugin/blob/c6ce7d1c2e8c65795971dd1b3634d67837f7add3/docs/commands-details/workload_get.md?plain=1#L87) as well

Reply:
okay, both updated